elements/pfe-switch/src/pfe-switch.html

```diff
@@ -0,0 +1,4 @@
+<div class="pfe-switch">
```

Comment:
I'm not sure this div is necessary.  You can style the `:host`, `<pfe-switch>`, directly in the stylesheet by using `:host` without needing to assign a class.  If you really want the class, in the JS you could add the class to the host element too.

Reply:
Styling tags makes me itchy... but you're right.. old habits, something something